clang/lib/Driver/ToolChains/HLSL.h

```diff
@@ -42,6 +42,19 @@ class LLVM_LIBRARY_VISIBILITY MetalConverter : public Tool {
                     const llvm::opt::ArgList &TCArgs,
                     const char *LinkingOutput) const override;
 };
+
+class LLVM_LIBRARY_VISIBILITY LLVMObjcopy : public Tool {
+public:
+  LLVMObjcopy(const ToolChain &TC)
+      : Tool("hlsl::LLVMObjcopy", "llvm-objcopy", TC) {}
```

Comment:
I don't think the `hlsl` is relevant here. It's really just objcopy.
```suggestion
      : Tool("LLVMObjcopy", "llvm-objcopy", TC) {}
```